core/src/main/resources/hudson/widgets/HistoryWidget/entry.jelly

```diff
@@ -40,7 +40,11 @@ THE SOFTWARE.
         <a class="tip model-link inside build-link display-name" update-parent-class=".build-row" href="${link}">${build.displayName}</a>
       </div>
       <div class="pane build-details" time="${build.timestamp.time.time}">
-        <a class="tip model-link inside build-link" href="${link}" update-parent-class=".build-row">
+        <j:set var="linkTitleAttr" value=""/> 
+        <j:if test="${!build.building}">
+          <j:set var="linkTitleAttr">${%Took} ${build.durationString}</j:set>
```

Comment:
The I18n string should probably be parameterized with `${build.durationString}` not all languages write it in that order.

Reply:
Some hint on how to do that?

Reply:
Here are some docs which are far from ideal: https://wiki.jenkins.io/display/JENKINS/Internationalization#Internationalization-Markingmessagesinjellyfiles